sway-lib-std/src/call_frames.sw

```diff
@@ -20,13 +20,47 @@ const SECOND_PARAMETER_OFFSET: u64 = 74;
 //  Accessing the current call frame
 //
 /// Get the current contract's id when called in an internal context.
+///
+/// # Additional Information
+///
 /// **_Note:_** If called in an external context, this will **not** return a contract ID.
-// @dev If called externally, will actually return a pointer to the transaction ID.
+/// If called externally, will actually return a pointer to the transaction ID.
+/// 
+/// # Returns
+///
+/// * [ContractId] - The contract id of this contract.
+///
+/// # Examples
+///
+/// ```sway
+/// use std::call_frames::contract_id;
+///
+/// fn foo() {
+///     let this_contract = contract_id();
+/// }
+/// ```
 pub fn contract_id() -> ContractId {
     ContractId::from(asm() { fp: b256 })
 }
 
 /// Get the `asset_id` of coins being sent from the current call frame.
+///
+/// # Returns
+/// 
+/// * [ContractId] - The asset included in the current call frame.
+///
+/// # Examples
+///
+/// ```sway
+/// use std::call_frames::msg_asset_id;
+/// use std::contsants::BASE_ASSET_ID;
```

Comment:
There should be a single use statement per lib, thats the generally followed convention. Also constants is mispelled
```suggestion
/// use std::{call_frames::msg_asset_id, constants::BASE_ASSET_ID};
```
Please remove the extra use statements in the rest of the code aswell